### PR TITLE
Add LTS 202406.00 release 

### PR DESCRIPTION
**Pull request by chinglee-iot**
**Description of changes:**
* Add LTS 202406.00 release 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by kar-rahul-aws** [comment]
CODEOWNERS file can be added to require PR-BR review before merging of a pull request.

File: README.md
Comment (sirnish) [inline README.md:9]: FreeRTOS 202210-LTS](https://github.com/FreeRTOS/FreeRTOS-LTS/tree/202210-LTS should be probably 202406

Comment (chinglee-iot) [reply README.md:9]: Update the version and remove the link here. Thanks for the catch.

**Comment by chinglee-iot** [comment]
> CODEOWNERS file can be added to require PR-BR review before merging of a pull request.

Thank you. Added in new commit.

**Review by aggarg — APPROVED** [review]

**Review by ActoryOu — APPROVED** [review]

**Pull request merged**
